### ISSUE TITLE
Alpine 3.3 broken with latest conf-which update

### ISSUE THREAD

**Issue by hcarty** [ocaml/opam-repository#9584]
#9568 adds a depext under Alpine for the `which` package.  That package does not exist under Alpine 3.3.  Does depext support OS version constraints by any chance?  If not, is Alpine 3.3 considered new enough to be supported?

If this is fixable I'd be happy to make a PR.

**Comment by dbuenzli** [comment]
As you discovered on https://github.com/ocaml/opam-depext/issues/43 `depexts` does currently not support OS version constraints.

**Comment by dbuenzli** [comment]
So it seems that `which` is simply available on stock alpine from 3.{3-6} (see below) but the package is only available since 3.4, see [here](https://pkgs.alpinelinux.org/packages?name=which&branch=&repo=&arch=&maintainer=). 

So I'd suggest that [this line](https://github.com/ocaml/opam-repository/blob/64e1b844d81315dcd47b907623d99f5866514d52/packages/conf-which/conf-which.1/opam#L10) should be commented with the above info until `depexts` support os versions.

Can you @hcarty or @UnixJunkie the maintainer of the `conf-which` package please make that change ? 

```
> docker run -it alpine:3.6 /bin/ash
/ # type which 
which is /usr/bin/which
> docker run -it alpine:3.5 /bin/ash
/ # type which 
which is /usr/bin/which
> docker run -it alpine:3.4 /bin/ash
/ # type which 
which is /usr/bin/which
> docker run -it alpine:3.3 /bin/ash
/ # type which 
which is /usr/bin/which
> docker run -it alpine:3.2 /bin/ash
/ # type which 
which is /usr/bin/which

**Comment by dbuenzli** [comment]
Closed by #9727

**Comment by hcarty** [comment]
Thank you @dbuenzli and @UnixJunkie!
